src/plugins/admonition/schema.js

```diff
@@ -36,7 +38,7 @@ export default {
         admonition: {
             data: {
                 type: v => TYPES.includes(v),
-                class: c => c == null || Array.isArray(c),
+                class: c => c == null || Immutable.List.isList(c),
```

Comment:
It would be good to also check whether classes themself are correct (i.e. they don't contain white spaces).